### PR TITLE
fix: UI bug fix

### LINKED ISSUE #1926: UI enhancement in speaker profile
**Actual Behaviour**
Overlapping of text and share button on the top right of speaker Profiles.

**Expected Behaviour**
There should be some padding of the textView so it doesn't Overlap.

**Steps to reproduce it**
Scroll down on a Speaker Profile. The about line underneath the name overlaps the share icon.

**Screenshots of the issue**
whatsapp image 2017-09-24 at 12 53 29 am

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by manki11**
Fixes #1926 

margin-right was increased so the textview no longer overlaps the share button.

Screenshots for the change: 
Before:
whatsapp image 2017-09-24 at 12 53 29 am

After:
whatsapp image 2017-09-24 at 1 40 27 am

### CODE REVIEW

File: android/app/src/main/gen/org/fossasia/openevent/BuildConfig.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/*___Generated_by_IDEA___*/
+
+package org.fossasia.openevent;
```
Comment (iamareebjamal) [inline android/app/src/main/gen/org/fossasia/openevent/BuildConfig.java]: Please remove these auto generated files

File: android/app/src/main/gen/org/fossasia/openevent/Manifest.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/*___Generated_by_IDEA___*/
+
+package org.fossasia.openevent;
+
+/* This stub is only used by the IDE. It is NOT the Manifest class actually packed into the APK */
+public final class Manifest {
```
Comment (iamareebjamal) [inline android/app/src/main/gen/org/fossasia/openevent/Manifest.java]: Please remove these auto generated files

File: android/app/src/main/gen/org/fossasia/openevent/R.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/*___Generated_by_IDEA___*/
+
+package org.fossasia.openevent;
```
Comment (iamareebjamal) [inline android/app/src/main/gen/org/fossasia/openevent/R.java]: Please remove these auto generated files

File: android/app/src/main/res/layout/activity_speakers.xml [hunk 4]
```diff
@@ -86,6 +86,7 @@
                     android:layout_height="wrap_content"
                     android:textColor="@android:color/white"
                     android:textSize="@dimen/text_size_large"
+                    android:layout_marginRight="40dp"
```
Comment (iamareebjamal) [inline android/app/src/main/res/layout/activity_speakers.xml]: Please use a value from predefined dimens

File: android/build.gradle [hunk 5]
```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.1'
+        classpath 'com.android.tools.build:gradle:2.3.3'
```
Comment (iamareebjamal) [inline android/build.gradle]: Please don't change values not related to the particular issue, this creates problems in tracking changes over time :smile:

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

**Comment by manki11** [comment]
@iamareebjamal Have made the changes.

**Comment by iamareebjamal** [comment]
Thanks, please also change commit message to refer to proper issue

**Comment by iamareebjamal** [comment]
Please use this format so it addresses which issue it fixes. 

fix: Add margin to overlapping Speaker subtitle

And please squash these commits

**Comment by manki11** [comment]
@iamareebjamal My first PR. Sorry for the inconvenience for some reason i am unable to squash the commits into one.

**Comment by iamareebjamal** [comment]
@manki11 Please follow this https://www.devroom.io/2011/07/05/git-squash-your-latests-commits-into-one/

Before this, run these commands

```
git fetch upstream development
git rebase upstream/development
```

**Comment by manki11** [comment]
@iamareebjamal did the squashing

**Comment by iamareebjamal** [comment]
> Please use this format so it addresses which issue it fixes.
> 
> fix: Add margin to overlapping Speaker subtitle

Awesome, please do this as last step

**Review by iamareebjamal — APPROVED** [review]

**Comment by mariobehling** [comment]
Thanks!

**Pull request merged**